woocommerce/woocommerce
# Fix cart widget in Safari when using browser back button

Fixes #21544

The no-store suggestion in #21544 did not work for me. Which I found out after fixing all the phpcs issues in ajax class (be05eea). Enjoy that one @claudiosanches 

The fix I did find was adding something unique to the fragment request. For that, I used time. 
34998ca

To test,

Open safari, shop page
Add to cart
Go to page 2
Add to cart. Note cart contents + total.
Use back button.
Ensure the cart shows the correct (new) total and contents.
